src/clang_complete.cc

```diff
@@ -165,10 +167,11 @@ std::unique_ptr<CompilerInstance> BuildCompilerInstance(
     CompletionSession &session, std::unique_ptr<CompilerInvocation> CI,
     DiagnosticConsumer &DC, const WorkingFiles::Snapshot &snapshot,
     std::vector<std::unique_ptr<llvm::MemoryBuffer>> &Bufs) {
-  StringRef main = CI->getFrontendOpts().Inputs[0].getFile();
+  auto main =
```

Comment:
CI->getFrontendOpts().Inputs[0].getFile() can be `"../src/stdio/asprintf.c"` because `compile_commands.json` main source file is not normalized now.

Reply:
Regarding 
"Add path normalization around clang::FrontendInputFile::getFile()"

I moved `ResolveIfRelative` to `utils.h` and use that to address the issue.